### ISSUE TITLE
Supporting Webpack 5

### ISSUE THREAD

**Issue by jasonwilliams** [Izhaki/nodemon-webpack-plugin#83]
Testing application with Webpack 5.0.0
Warning emitted
```
warning " > nodemon-webpack-plugin@4.1.1" has incorrect peer dependency "webpack@4".
```

Would be good to have peerDependencies set to
"webpack": " ^4.0.0 || ^5.0.0"

https://github.com/webpack/changelog-v5/blob/master/MIGRATION%20GUIDE.md

**Comment by Izhaki** [comment]
Uuuuuuhhh! Lot's of work for me?

Does it work otherwise?

(Updating peer dependencies is easy - adding the tests will take a bit longer!)

**Comment by jasonwilliams** [comment]
Seems to work ok

**Comment by Izhaki** [comment]
That's odd. Tests are failing and it seems to be for a good reason (https://github.com/webpack/webpack/issues/9802#issuecomment-553115900).

**Comment by jasonwilliams** [comment]
Sorry about that im wondering if i tested against v4 instead of 5.

**Comment by Izhaki** [comment]
With tiny code changes, we now have Webpack 5 support. Landed as 4.2.0.

Thanks for flagging this!

**Comment by TheLarkInn** [comment]
Thanks for this early support! 🥰
